pkg/helm/portforwarder/portforwarder.go

```diff
@@ -34,13 +34,12 @@ var (
 )
 
 // New creates a new and initialized tunnel.
-func New(namespace string, client kubernetes.Interface, config *rest.Config) (*kube.Tunnel, error) {
+func New(namespace string, port int, client kubernetes.Interface, config *rest.Config) (*kube.Tunnel, error) {
```

Comment:
Technically this is a breaking change from the SDK's standpoint, and we try to avoid those. Would you be willing to change this back so existing users can consume `New()` the same way as before, but perhaps introduce a new function called `NewWithPort()`?

Reply:
Of course!